FrontEnd/src/components/basics/RadioGroup/RadioButton/RadioButton.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FunctionComponent, useState } from 'react';
+import styles from './RadioButton.module.scss';
+import { Body1 } from '@basics';
+import { v4 as uuidv4 } from 'uuid';
+
+export interface RadioButtonProps {
+  id?: string;
+  value: string; // required for value extracting
+  withLabel?: boolean; // used when we want to add text on the side
+  name: string;
+  // TODO add onChange here when actually coding the wizard form
+}
+
+const RadioButton: FunctionComponent<RadioButtonProps> = ({
+  id,
+  value,
+  withLabel,
+  name,
+}) => (
+  <div className="d-flex">
+    <input
+      type="radio"
+      className={styles.radioInput}
+      name={name}
+      value={value}
+    />
+    {withLabel && (
+      <label htmlFor={id || uuidv4()}>
```

Comment:
I don't think there's a use for "htmlFor" if the id is not set in the input as well. You should probably create the id above the div and then use it as the id for the input and the htmlFor value too.